prow/config.yaml

```diff
@@ -426,8 +426,33 @@ tide:
     - do-not-merge/hold
     - do-not-merge/invalid-owners-file
     - do-not-merge/work-in-progress
+  # Auto-merge PRs generated for automatic configs release by robot.
+  - repos:
+    - bazelbuild/bazel-toolchains
+    labels:
+    - automerge
+    missingLabels:
+    - do-not-merge
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+  - repos:
+    - bazelbuild/bazel-toolchains
+    - bazelbuild/rules_docker
+    - bazelbuild/rules_k8s
```

Comment:
Please combine this query with the query for `GoogleCloudPlatform/netd` to save API rate limit tokens.

Reply:
Done